service/service.go

```diff
@@ -96,12 +87,6 @@ func GrpcConnection(client, server string, insecureAllowed bool) (*grpc.ClientCo
 
 		// TODO(waigani) check error
 
-		// if cert is stale, get a new one
```

Comment:
this whole `if err != nil` block starting at :83 can just be `return nil, errors.Trace(err)` – `cert` is always nil